packages/components/input-number/src/index.vue

```diff
@@ -249,6 +250,17 @@ export default defineComponent({
       }
       data.userInput = null
     }
+
+    const focus = () => {
+      nextTick(() => {
```

Comment:
Why use `nextTick`?

Reply:
It was the reason for this bug at the time. https://github.com/ElemeFE/element/issues/18573

Reply:
But this is `input-number`, it seems that this problem will never have. I will update it.